### PR TITLE
fix fire-pos

### PR DESCRIPTION
**Pull request by millay1114**
I fix to fire-pos.
But, I think it is not well. because it is very redundancy.

Please comment to me how do I fix this code 💦

### CODE REVIEW

File: app/game/field.py [hunk 1]
```diff
@@ -153,24 +195,27 @@ def fire_bomb(self, point):
...
                     person.dead = True
                     put_object_to_field(self.bombs, fire_pos, FiredPerson(person))
-            object = field_object(self.objects, fire_pos)
-            if object == Object.block:
+            object_at_fire = field_object(self.objects, fire_pos)
+            if object_at_fire == Object.block:
                 put_object_to_field(self.objects, fire_pos, Object.empty)
+            item_at_fire = field_object(self.items, fire_pos)
+            if isinstance(item_at_fire, Item):
+                put_object_to_field(self.items, fire_pos, None)
```
Comment (kouki-dan) [inline app/game/field.py]: It may break all item below brock, isn't it?

File: app/game/field.py [hunk 2]
```diff
@@ -43,15 +43,57 @@ def __init__(self, owner, fire_count):
...
+            fire_points += [
+                fire_point,
+            ]
+            obj = field.get_field_object(point.diff(fire_point.x, fire_point.y))
+            if obj == Object.wall:
+                fire_points.pop()
+                break
+            if obj == Object.block:
+                break
+
```
Comment (kouki-dan) [inline app/game/field.py]: If you want to reduce for loop, you can write like this.

```
for x,y  in [[0, -1], [0, 1], [1, 0], [-1, 0]]:
    for i in range(1, self.fire_count+1):
        fire_point = Point(x*i, y*i)
....
```

Comment (kouki-dan) [inline app/game/field.py]: I think it may work, but I didn't confirm it.

File: app/game/field.py (hunk 1 above)
Comment (millay1114) [inline app/game/field.py]: You right.
I add ``elif`` to this line.

**Comment by millay1114** [comment]
i fixed it.

**Comment by millay1114** [comment]
fix!

**Pull request merged**